inflation-resources/tests/dataset/test_reader.py

```diff
@@ -0,0 +1,51 @@
+import pytest
+import datetime
+from inflation.dataset.reader import InflationJSONA101DatasetReader
+from pathlib import Path
+
+PRODUCT_PAGE_DATA_PATH = (
+    Path(__file__).parents[3]
+    / "inflation-resources/tests/data/product_page.json.gz"
+)
+
+
+@pytest.fixture(scope="function")
+def inflation_data_reader():
+    return InflationJSONA101DatasetReader()
+
+
+def test_read_product(
+    inflation_data_reader,
+):  # Q: 3 tane yok muydu niye 2 tane geldi?
+    record_data = inflation_data_reader.read(PRODUCT_PAGE_DATA_PATH)
+    results = [
```

Comment:
`truths` or something would be a better name.

Reply:
I changed as PRODUCT_PAGE_TRUTHS_PATH

Reply:
Sorry, you meant results. Okay now I did it, correctly.